### DISCUSSION TITLE
Renovate cannot handle AWS Aurora version numbers when using aws-rds datasource

### DISCUSSION THREAD

**Discussion by patrickpichler** [renovatebot/renovate#19928] [Request Help] [answered]
### Problem
I have a terraform configuration managing some AWS aurora RDS clusters. I configured the aws-rds datasource accoring to the [documentation](https://docs.renovatebot.com/modules/datasource/aws-rds/). 

When renovate bot runs, it prints the following debug log:
```
DEBUG: Dependency default has unsupported value 8.0.mysql_aurora.3.01.0
```

**Comment by rarkins** [comment, marked as answer]
You can customize versioning, either by switching to "loose", or by using extractVersion to extract out only the parts you care about

**Reply by patrickpichler** [reply]
I didn't think about that! Thank you for the fast response :slightly_smiling_face:

**Comment by phyzical** [comment]
i just ran into this, figured i would add what i did to solve it, 
This is terraform + rds datasource + aurora-mysql
simply because i wasted so much time to realise the default versioning of rds datasource isn't compatible with aurora

```
 regexManagers: [
      {
        "description": "Update RDS",
        "customType": "regex",
        "fileMatch": [".+\\.tf$"],
        "matchStrings": [
          "\\s*#\\s*renovate:\\s*amiFilter=(?<lookupName>.+?) depName=(?<depName>.*) versioning=(?<versioning>.*)\\s*.*_version\\s*=\\s*\"(?<currentValue>.*)\""
        ],
        "datasourceTemplate": "aws-rds"
      },
  ]
```

```
# renovate:amiFilter=[{"Name":"engine","Values":["aurora-mysql"]},{"Name":"engine-version","Values":["8.0"]}] depName=aurora-mysql versioning=loose
engine_version = "8.0.mysql_aurora.3.05.2"
```

**Reply by HonkingGoose** [reply]
Thank you for the comment. I'm trying to get your example in the docs with this PR:

- #29870

I'm not sure if the maintainers want the example, so it may get stripped out during the review process.

**Reply by phyzical** [reply]
its all good, was more for anyone who stumbles here 👍
